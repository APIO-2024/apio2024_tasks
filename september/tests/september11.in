txy4h26c1rm1uv8tr3eonahd67u8h56x
4
7 5
0 1 1 2 3 4
6 5 4 2 1 3
6 5 4 2 1 3
6 5 4 2 3 1
6 5 4 2 1 3
6 4 5 2 3 1
10 5
0 0 2 3 3 4 6 6 8
1 9 7 5 6 8 3 2 4
9 1 7 5 8 6 2 4 3
1 7 9 8 6 2 5 3 4
1 9 8 7 6 2 5 3 4
9 8 7 1 6 5 3 4 2
7 5
0 1 2 2 4 5
3 5 6 4 2 1
5 3 6 4 2 1
5 6 4 3 2 1
5 4 6 3 1 2
5 6 4 3 1 2
6 5
0 1 1 2 3
2 5 3 4 1
2 5 3 1 4
5 3 1 4 2
3 5 1 4 2
5 3 4 2 1
