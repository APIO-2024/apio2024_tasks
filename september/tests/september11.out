7ckgnn4wyi495puj3ibqf81dqvapyv6b
4
1
2
1
